sdk/textanalytics/ai-text-analytics/samples/javascript/analyzeSentimentWithOpinionMining.js

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * This sample demonstrates how to analyze sentiment on a more granular level,
+ * mining individual opinions from reviews (also known as aspect-based 
+ * sentiment analysis). This feature is only available for clients with api 
+ * version v3.1-preview.1.
+ * 
+ * In this sample, we will be a customer who is trying to figure out whether 
+ * they should stay at a specific hotel. We will be looking at which aspects of 
+ * the hotel are good, and which are not.
+ */
+
+const { TextAnalyticsClient, AzureKeyCredential } = require("@azure/ai-text-analytics");
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
```

Comment:
need to use `require` rather than `import` for plain JS samples

Reply:
done.